src/main/java/mate/model/Driver.java

```diff
@@ -5,6 +5,8 @@
 public class Driver {
```

Comment:
I think we should define `toString()`. method for `Driver` to display info in exception messages